src/inter_dc_repl_vnode.erl

```diff
@@ -91,8 +91,7 @@ handle_command(trigger, _Sender, State=#state{partition=Partition,
                 [_H|_T] ->
                     case inter_dc_communication_sender:propagate_sync(
                            {replicate, Transactions}, DCs) of
-                        ok -> NewReaderState;
-                        _  -> Reader
+                        ok -> NewReaderState
```

Comment:
Could the missing `_` case here generate a `case_clause` error and crash sometimes?